package.json

```diff
@@ -19,5 +20,9 @@
   "dependencies": {
     "express": "^4.16.3",
     "jsonwebtoken": "^8.3.0"
+  },
+  "devDependencies": {
+    "ts-node": "^7.0.1",
```

Comment:
Move both to dependencies.
Otherwise, the change you made to make it run typescript on production would break.